src/BidiStream.php

```diff
@@ -69,6 +70,7 @@ public function __construct(
             $this->resourcesGetMethod = $streamingDescriptor['resourcesGetMethod'];
         }
         $this->logger = $logger;
+        $this->identifier = spl_object_id($this) . rand(1000,9999);
```

Comment:
using the `.` will convert both ints to a string. I think that's okay, but we'll need to change the type of `$identifier` to be `string`

Reply:
This has been updated 🙏 We are calculating the `requestId` different now.